### PR TITLE
WarriorsguildV2 

### PR DESCRIPTION
**Pull request by Lajnux**
### Description:

Removed most unnecessary activities and stuck with main activity.

Now can only do animated armours and cyclops

(Need to add Cyclops to OldschoolJS)

### Changes:

Removed most unnecessary activities and stuck with main activity.

Now can only do animated armours and cyclops

-   [] I have tested all my changes thoroughly.

### CODE REVIEW

File: src/tasks/minions/minigames/warriorsGuild/cyclopsActivity.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+		if (!bankHasItem(userBank, defenders[defenders.length - 1].itemID, 1)) {
+			for (let i = 0; i < quantity; i++) {
+				if (roll(defenders[defenders.length - 1].rollChance)) {
+					loot = addItemToBank(loot, defenders[defenders.length - 1].itemID);
+				}
+			}
+		}
+		// Remove Catacomb totem pieces
+		loot = removeItemFromBank(loot, 19677, user.numItemsInBankSync(19677));
+		loot = removeItemFromBank(loot, 19679, user.numItemsInBankSync(196779));
```
Comment (kevinsmendes) [inline src/tasks/minions/minigames/warriorsGuild/cyclopsActivity.ts:84]: You probably didn't mean itemID 196779

**Review by kevinsmendes — CHANGES_REQUESTED** [review]
You need to add `Activity.Cyclops` and `Activity.AnimatedArmour` to `tickerTaskFromActivity.ts` to allow the user to cancel the trip. Also, would be good to allow the user to not waste time by only thinking they have 1 piece of each armor. The bank is right around the animated armor location and wasting up to 40 minutes because it broke on the first armor will be pissing people off. I recommend you just keep breaking pieces as long the user have them in the bank.

File: src/commands/Minion/warriorsguild.ts [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+				return msg.send(response);
+			}
+			throw `That isn't a valid animated armour tier to kill, the available tiers are: ${Armours.map(
+				tier => tier.name
+			).join(', ')}. For example, \`${msg.cmdPrefix}warriorsguild animation 5 bronze\``;
+		}
+
+		if (minigame === 'cyclops') {
+			// Check if either 100 warrior guild tokens or attack cape (similar items in future)
+			if (!bankHasItem(userBank, 8851, 100) && !bankHasItem(userBank, 9747, 1)) {
```
Comment (kevinsmendes) [inline src/commands/Minion/warriorsguild.ts:156]: This should `if (!bankHasItem(userBank, 8851, 100) || !bankHasItem(userBank, 9747, 1)) {`.

**Comment by Lajnux** [comment]
merged

**Pull request closed without merging**